heka/sandbox/decoders/extract_executive_summary.lua

```diff
@@ -110,6 +110,19 @@ local function get_os_version()
 end
 
 
+local function get_profile_creation()
+    local json = read_message("Fields[environment.profile]")
+    local ok, profile = pcall(cjson.decode, json)
+    if not ok then return 0 end
+
+    local days = profile.creationDate
+    if type(days) == "number" and days > 0 then
+        return days * SEC_IN_DAY * 1e9
```

Comment:
Ah, I see it gets reset [here](https://github.com/mozilla-services/data-pipeline/blob/profile_creation/heka/plugins/fx/executive_report.c#L396). I thought it would report as "new" once per period. Never mind.